starboard/shared/starboard/player/filter/filter_based_player_worker_handler.cc

```diff
@@ -92,6 +93,35 @@ FilterBasedPlayerWorkerHandler::FilterBasedPlayerWorkerHandler(
   update_job_ = std::bind(&FilterBasedPlayerWorkerHandler::Update, this);
 }
 
+FilterBasedPlayerWorkerHandler::FilterBasedPlayerWorkerHandler(
+    const StarboardExtensionEnhancedPlayerCreationParam*
+        starboard_extension_enhanced_player_creation_param,
+    SbDecodeTargetGraphicsContextProvider* provider)
```

Comment:
Consider using constructor delegation to simplify the implementation.

Reply:
Done.